### ISSUE TITLE
WP cleanup

### ISSUE THREAD

**Issue by fortunac** [draperlaboratory/cbat_tools#263]
With our upgrade to bap 2.2.0, we should also clean up WP:

- Rename `bap_wp` to `bap-wp`
- Rename `master` to `main`

**Comment by codyroux** [comment]
Have we moved forward on this?

**Comment by fortunac** [comment]
We have not started this. @jtpaasch are there any obstacles preventing us from going ahead with making this change?

**Comment by jtpaasch** [comment]
Nope! I made this change in our VIBES project, and it was painless. Have a look at the VIBES repo, in the `bap-vibes` folder. Check out the `bap-vibes.opam` file (especially the `make` bit...), the `Makefile` there (notice we don't need to do `dune install`...we only do `opam install`), and the `dune` file inside `src`.
